src/components/ENSListPage/ENSListPage.tsx

```diff
@@ -89,8 +89,18 @@ export default class ENSListPage extends React.PureComponent<Props, State> {
     )
   }
 
+  renderUseAsAliasButton(ens: ENS, hasProfileCreated: boolean) {
+    return (
+      <div className="popup-button">
+        <Button className="ui basic button" onClick={() => this.handleOpenModal(ens.name)} disabled={!hasProfileCreated}>
+          {t('ens_list_page.button.use_as_alias')}
+        </Button>
+      </div>
+    )
+  }
+
   renderEnsList() {
-    const { ensList } = this.props
+    const { ensList, hasProfileCreated } = this.props
```

Comment:
```suggestion
    const { ensList, avatar } = this.props
    const hasProfileCreated = !!avatar
```
what do you think about this alternative? so we keep the prop as the selector